build-docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e # exit with nonzero exit code if anything fails
+
+# install graphviz to generate diagrams
+sudo apt-get update -qq
+sudo apt-get install -qq graphviz
```

Comment:
This isn't necessary; see http://docs.travis-ci.com/user/apt/ for how to add packages without having to reenable `sudo`.